src/main/java/de/tum/cit/aet/artemis/buildagent/dto/BuildAgent.java

```diff
@@ -0,0 +1,10 @@
+package de.tum.cit.aet.artemis.buildagent.dto;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+public record BuildAgent(String name, String memberAddress, String displayName) implements Serializable {
```

Comment:
Why do all these DTOs not contain `DTO` in their name? I confused them for actual domain objects for a few seconds
```suggestion
public record BuildAgentDTO(String name, String memberAddress, String displayName) implements Serializable {
```